content/fr/_index.md

```diff
@@ -57,7 +57,7 @@ performance et le comportement de vos logiciels.
 
 {{% blocks/section color="dark" type="row" %}}
 
-{{% blocks/feature icon="fas fa-chart-line" title="Traces, Métriques, Logs" url="docs/concepts/observability-primer/" %}}
+{{% blocks/feature icon="fas fa-chart-line" title="Traces, Métriques, Logs" %}}
```

Comment:
Why this change? It is forcing a difference between the `fr` and `en` pages (which shouldn't be necessary) and invalidating the `default_lang_commit` value.

Reply:
@chalin : I was struggling to solve the link issue in the _index.md. I checked how other languages handled this and noticed that both the Spanish and Portuguese versions removed this URL.